stubs/braintree/braintree/environment.pyi

```diff
@@ -1,33 +1,38 @@
-from typing import Any
+from typing import Any, ClassVar
 
 from braintree.exceptions.configuration_error import ConfigurationError as ConfigurationError
 
 class Environment:
-    __name__: Any
+    Development: ClassVar[Environment]
+    QA: ClassVar[Environment]
+    Sandbox: ClassVar[Environment]
+    Production: ClassVar[Environment]
+    All: ClassVar[dict[str, Environment]]
+    __name__: str
     is_ssl: Any
     ssl_certificate: Any
```

Comment:
To get rid of the `Any`s (which should be `Incomplete`s):

```suggestion
    is_ssl: bool
    ssl_certificate: bool | str | None
```

https://github.com/braintree/braintree_python/blob/a05909893e332879297dbf560431c8ae32aa657b/tests/unit/test_environment.py#L38C1-L42

`ssl_certificate` is passed to request's `send()` method, which takes `bool | str | None` as `verify` argument:

https://github.com/braintree/braintree_python/blob/a05909893e332879297dbf560431c8ae32aa657b/braintree/util/http.py#L111-L131

Reply:
Hmmm, just because it's passed to `send()` doesn't mean that `bool | str | None` is the correct type, though -- maybe it's always a `str`, in which case `bool | str | None` will cause false positives. For now, I think I'll just use `Incomplete` for `ssl_certificate`